modules/tasks/taskInvite.js

```diff
@@ -14,7 +14,7 @@ module.exports = Promise.method(function ({ id }, { message, email }) {
       i18n.setLocale(language)
       SendMail.success(
         { email, language },
-        i18n.__('mail.invite.send.action'),
+        i18n.__('mail.invite.send.action', { name: name }),
         `${i18n.__('mail.invite.send.message', {
           title: task.title,
           url: taskUrl,
```

Comment:
@spikeboom we should add the template on the message body as well

Reply:
solved